oar/notificator/jira_notificator.py

```
@@ -71,11 +79,60 @@ class NotificationService:
+        Create GitHub App Reader client from environment variables.
+
+        Returns:
+            GitHubApp instance, or None if credentials are missing or initialization fails.
+        """
+        app_id = os.environ.get(ENV_VAR_GITHUB_APP_READER_ID)
+        private_key_path = os.environ.get(ENV_VAR_GITHUB_APP_READER_PRIVATE_KEY)
+        if not app_id or not private_key_path:
+            return None
+        try:
+            return GitHubApp(app_id, private_key_path)
+        except Exception as e:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)